CHANGES.txt

```diff
@@ -126,3 +126,6 @@ Changes
 
 Fixes
 =====
+
+- Added support for subscript expressions on the ``values`` column of
```

Comment:
this sounds a bit like a feature not a fix. but maybe that's my personal issue, leave it up to you.